src/Core/Domain/Customer/Group/Command/AddCustomerGroupCommand.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace PrestaShop\PrestaShop\Core\Domain\Customer\Group\Command;
+
+use PrestaShop\PrestaShop\Core\Domain\Customer\Group\ValueObject\GroupName;
+
+class AddCustomerGroupCommand
+{
+    /** @var string[] */
+    private $localizedNames;
+    /** @var float */
+    private $reduction;
+    /** @var int */
+    private $priceDisplayMethod;
+    /** @var bool */
+    private $showPrice;
+
+    /**
+     * @param string[] $localizedNames
+     * @param float $reduction
+     * @param int $priceDisplayMethod
+     * @param bool $showPrices
+     */
+    public function __construct(
+        array $localizedNames,
+        float $reduction,
+        int $priceDisplayMethod,
```

Comment:
I see that valid values for this are either TAX_INCL or TAX_EXCL so I think we could have bool here like
```suggestion
        bool $displayPriceTaxIncluded,
```

Reply:
Indeed, the DB uses an int but the possible values are only 0 or 1 https://github.com/PrestaShop/PrestaShop/blob/07688172a6aa78cb9e925edf201ab2e83eead717/classes/Group.php#L76-L77

It may be worth improving the model in the command, even if we don't change the ObjectModel or the DB structure, it's a first step to improving our model later as well if we don't find any valid usage for this int value.

Reply:
```suggestion
        int $displayPriceTaxExcluded,
```
Although be careful the `1` value matches with price tax excluded, so we should "revert" the naming so that `true` matches the behaviour without taxes

Reply:
I went with `bool $displayPriceTaceExcluded` to match the values Jonathan linked, I saw that too, so that `true == 1 == PRICE_DISPLAY_METHOD_TAX_EXCL` & `false == 0 == PRICE_DISPLAY_METHOD_TAX_INCL`